connectivity/FEATURE_BLE/libraries/TARGET_CORDIO_LL/stack/controller/sources/common/chci/chci_tr.c

```diff
@@ -369,15 +366,12 @@ static void chciTxComplete(void)
 
   if (blkLen)
   {
-  #if (CHCI_TR_UART == 1)
     PalUartWriteData(PAL_UART_ID_CHCI, chciTrCb.pWrBuf + chciTrCb.wrBufOffs, blkLen);
-  #endif
     chciTrCb.wrBufOffs += blkLen;
   }
   else
   {
-    /* Completion handled in task context, i.e. ChciTrService(). */
-    chciTrCb.wrBufComp = TRUE;
+    chciTrSendComplete();
```

Comment:
this needs the CHCI_TR_UART guards as it's not needed in our transport as there is nothing actually being sent, the message is just passed on

Reply:
Like this?
```
  #if (CHCI_TR_UART == 1)
    /* Completion handled in task context, i.e. ChciTrService(). */
    chciTrCb.wrBufComp = TRUE;
  #else
    chciTrSendComplete();
  #endif
```

Reply:
```suggestion
  #if (CHCI_TR_UART == 1)
    chciTrSendComplete();
  #endif
```

Reply:
wrBufComp has been removed